tests/Extensions/Hosting.Services.Web.UnitTests/HostBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Fabric;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Omex.Extensions.Hosting.Services;
+using Microsoft.Omex.Extensions.Hosting.Services.Web;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ServiceFabric.Mocks;
+
+namespace Hosting.Services.Web.UnitTests
+{
+	[TestClass]
+	public class HostBuilderExtensionsTests
+	{
+		[TestMethod]
+		public void BuildStatelessService_TypesRegistredStateless() =>
```

Comment:
Typo in `Registered`. Applies throughout this file

Reply:
I will resolve it in the follow-up pr https://github.com/microsoft/Omex/pull/168